README.md

```diff
@@ -1,7 +1,9 @@
 ## Test262 is the official ECMAScript (ECMA-262) conformance suite.
 
+Test262 tests conformance to the continually maintained draft future ECMAScript standard found at http://tc39.github.io/ecma262/ , together with any Stage 3 or later TC39 proposals. It is maintained by Gorkem Yakin (@goyakin) with significant contributions from many in the ECMAScript community.
+
```

Comment:
We, for one, would appreciate the ability to get conformance data against a particular version of the spec :) I think maintaining a front-matter attribute isn't a bad idea for this purpose. It's even useful information in other cases (like, answering the question "when was this behavior introduced??") However, it won't be perfect as tests may be removed when made invalid by later versions. This seems good.